### PR TITLE
Show template context on included templates

### LINKED ISSUE #902: SQL panel points a query in an included template to the same line of its parent template
Tested with:

views.py
````python
from test_debug_toolbar.models import Question

def index(request):
    return render(request, 'index.html', {'questions': Question.objects.all()})
````

index.html
````html
{% extends 'base.html' %}

{% block content %}

<h5>Hello there! This is a test for debug toolbar.</h5>

<p>Below is an included template that makes a SQL query. But in debug toolbar, it is signified in "index.html", while it should be in "inner.html".</p>

{% include 'inner.html' %}

{% endblock content %}
````

inner.html
````html
<p>Here's inner.html:</p>


<p>All questions:</p>
{{ questions }}
````

The SQL query of `Question.objects.all()` is executed at the line 5 of `inner.html`, but in the SQL panel, it is incorrectly pointed to the line 5 of its parent template `index.html`:

image

Tested with current master and develop branch, and Django 1.10.4.

For the complete test Django project: https://github.com/lingxiaoyang/test_debug_toolbar

### PR DESCRIPTION
**Pull request by gilmrjc**
This PR makes possible to see the template context of a nested template on the SQL panel.

Fix #902

### CODE REVIEW

**Review by auvipy — APPROVED** [review]

**Review by auvipy — CHANGES_REQUESTED** [review]
can you please check the sqlite failures?

**Comment by matthiask** [comment]
The test failures are probably related to https://github.com/jazzband/django-debug-toolbar/pull/1427 

They are not related to sqlite3 – the problems appear in the sqlite3 build only because sqlite3 is the only build which runs tests using Django@master.

File: tests/templates/sql/nested.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+  {% include "sql/included.html" %}
```
Comment (tim-schilling) [inline tests/templates/sql/nested.html:3]: How come you went with an `include` here rather than using `{{ users }}`?

Comment (gilmrjc) [reply tests/templates/sql/nested.html:3]: From the issue #902 

> The SQL query of Question.objects.all() is executed at the line 5 of inner.html, but in the SQL panel, it is incorrectly pointed to the line 5 of its parent template index.html

The problem is that the SQL panel does not show the template context when using included templates. This file to test the correct behavior.

Comment (tim-schilling) [reply tests/templates/sql/nested.html:3]: I'm sorry, I didn't realize we were in the tests directory.

**Review by tim-schilling — APPROVED** [review]
Looks good. I ran this locally with some changes to the example app.

**Comment by gilmrjc** [comment]
I rebased the branch to include the new changes.

**Comment by matthiask** [comment]
Thanks!

**Pull request merged**